### PR TITLE
feat: add blog post story

### PR DESCRIPTION
**Pull request by edvicaty**
ticket: GAT-55 https://app.clickup.com/t/gwyur3

- Added Blog Post Story using the following components:  
- Contact,
- Header,
- Section,
- Image,
- BlogHead,
- Text,
- Link,
- Logo,
- Footer,
- MobileMenu,
- Heading

image
image
image
image

### CODE REVIEW

File: packages/gatsby-theme/src/pages/blog-page.stories.js
Comment (luisduenas) [inline packages/gatsby-theme/src/pages/blog-page.stories.js]: Let's use `ContactBlock` instead of the component, same for the other blocks, (`BlogHeadBlock`, `FooterBlock`, `HeaderBlock`)

**Comment by edvicaty** [comment]
On the refactor of the Footer component we forgot to remove the highlighted data on the FooterBlock story, this data is no longer used: 
image
I fixed this on the last commit @luisduenas

File: packages/gatsby-theme/src/pages/blog-page.stories.js
Comment (dinartecc) [inline packages/gatsby-theme/src/pages/blog-page.stories.js]: We should wait for the #35 PR to be merged before merging this one. Luis added an alias for the blocks.

File: packages/gatsby-theme/src/pages/blog-page.stories.js
Comment (dinartecc) [inline packages/gatsby-theme/src/pages/blog-page.stories.js]: Just to maintain our conventions, we should change this to `__css`.

File: packages/gatsby-theme/src/pages/blog-page.stories.js
Comment (georch) [reply packages/gatsby-theme/src/pages/blog-page.stories.js]: @edvicaty The PR #35  was merged to `master`, so please proceed to update this code.

Comment (edvicaty) [reply packages/gatsby-theme/src/pages/blog-page.stories.js]: done @georch

**Review by georch — APPROVED** [review]
LGTM

**Pull request merged**
